alacritty.yml

```diff
@@ -450,9 +458,11 @@ alt_send_esc: true
 #   - ClearLogNotice
 #   - SpawnNewInstance
 #   - ToggleFullscreen
-#   - ToggleSimpleFullscreen # macOS only
 #   - None
 #
+# Values for `action` (macOS only):
+#   - ToggleSimpleFullscreen: fullscreens without occupying another space
```

Comment:
I think using `fullscreens` as a verb doesn't sound great, `Enters fullscreen` sounds better to me I think.